.pipelines/release.yml

```diff
@@ -20,429 +23,423 @@ parameters:
+          displayName: Audit WinAppSDK applications path asset conflicts
+          inputs:
+            filePath: '.pipelines/verifyPossibleAssetConflicts.ps1'
+            arguments: -targetDir '$(build.sourcesdirectory)\$(BuildPlatform)\$(BuildConfiguration)\WinUI3Apps'
+            pwsh: true
+
+      #### MAIN SIGNING AREA
+      # reference https://dev.azure.com/microsoft/Dart/_git/AppDriver?path=/ESRPSigning.json&version=GBarm64-netcore&_a=contents for winappdriver
+      # https://dev.azure.com/microsoft/Dart/_git/AppDriver?path=/CIPolicy.xml&version=GBarm64-netcore&_a=contents
+
+        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
+          displayName: Sign Core PT
+          inputs:
+            ConnectedServiceName: 'Terminal/Console/WinAppDriver Team Code Signing Connection'
+            FolderPath: '$(BuildPlatform)/$(BuildConfiguration)' # Video conf uses x86 and x64.
+            signType: batchSigning
+            batchSignPolicyFile: '$(build.sourcesdirectory)\.pipelines\ESRPSigning_core.json'
+            ciPolicyFile: '$(build.sourcesdirectory)\.pipelines\CIPolicy.xml'
+
+        - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@3
+          displayName: Sign x86 directshow VCM
+          inputs:
+            ConnectedServiceName: 'Terminal/Console/WinAppDriver Team Code Signing Connection'
+            FolderPath: 'x86/$(BuildConfiguration)' # Video conf uses x86 and x64.
+            signType: batchSigning
+            batchSignPolicyFile: '$(build.sourcesdirectory)\.pipelines\ESRPSigning_vcm.json'
+            ciPolicyFile: '$(build.sourcesdirectory)\.pipelines\CIPolicy.xml'
+      #### END SIGNING 
+      ## END MAIN
+
+        - pwsh: |-
+            Move-Item msbuild.binlog "$(Build.ArtifactStagingDirectory)/"
+          displayName: Stage binlog into artifact directory
+          condition: always()
+
+        - task: ComponentGovernanceComponentDetection@0
+          displayName: Component Detection
+
+        - task: CopyFiles@2
+          displayName: Copying files for symbols
+          inputs:
+            contents: >-
+              **/*.pdb
+            flattenFolders: True
+            targetFolder: $(Build.ArtifactStagingDirectory)/Symbols-$(BuildPlatform)/
+         
+        - task: PowerShell@2
+          displayName: 'Remove unneeded files from ArtifactStagingDirectory'
+          inputs:
+            targetType: 'inline'
+            script: |
+              cd $(Build.ArtifactStagingDirectory)/Symbols-$(BuildPlatform)/
+              Remove-Item vc143.pdb
+              Remove-Item *test*
+
+        - task: PublishSymbols@2
+          displayName: Publish symbols path
+          continueOnError: True
+          inputs:
+            SearchPattern: |
+              $(Build.ArtifactStagingDirectory)/Symbols-$(BuildPlatform)/**/*.*
+            IndexSources: false
+            SymbolServerType: TeamServices
+            
+        - template: .pipelines/installer-steps.yml@self
```

Comment:
If I'm reading properly, according to [Reference Template Paths](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops&pivots=templates-includes#reference-template-paths) this SHOULD cause a problem since we're already in the `.pipelines` directory? But I'm not sure it actually does.

Reply:
Actually, never mind. I see the build success. Disregard :).

Reply:
This is a weird one. This step list is being transcluded inside a template from a different repository! I learned through trial and error that you need to add `@self` to get back to the originating repository... and that doing so removes any preexisting relative path. The template anchors inside the transcluded step list start out at the repo root 😄